templates/attributes.j2

```diff
@@ -2,6 +2,11 @@
 
 {% block title %}Attributes{% endblock %}
 
+{%- block metadesc -%}
+  {{ "Attributes are a tool for associating information with declarations. 
+  In the simplest case, an attribute is a tag that can be applied to a declaration. simp is a common example of this." | plaintext_summary }}
```

Comment:
Is there a reason to do `plaintext_summary` here?  Wouldn't it be easier to write the result directly?

Reply:
> Looks great! There is no way to make https://leanprover-community.github.io/mathlib_docs/data/complex/exponential.html#complex.exp' show the declaration doc string, right?

Thanks 😃 ! Unfortunately not - the meta tags have to be per page, not per anchor tag

Reply:
No real reason - just that it does the truncating to < 200 characters without me having to count hehe. Happy to change it to just putting the summary in directly

Reply:
Ok, if we need to respect hard character limits then it's ok.